### PR TITLE
Release PR for 2.5.2

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by jfy133** [comment]
@nf-core-bot fix linting

File: nextflow.config [hunk 1]
```diff
@@ -185,7 +186,7 @@ params {
     // Schema validation default options
     validationFailUnrecognisedParams = false
     validationLenientMode            = false
-    validationSchemaIgnoreParams     = 'genomes,igenomes_base'
+    validationSchemaIgnoreParams     = 'genome,genomes,igenomes_base'
     validationShowHiddenParams       = false
     validate_params                  = true
```
Comment (maxibor) [inline nextflow.config:191]: This should set to false according to https://nf-co.re/docs/contributing/pipeline_release_review_guidelines

**Comment by maxibor** [comment]
Looks good, just one comment (according to https://nf-co.re/docs/contributing/pipeline_release_review_guidelines): some contributors are missing in the README (at least @CarsonJM as far as I can see)

**Review by maxibor — APPROVED** [review]

File: nextflow.config (hunk 1 above)
Comment (maxibor) [reply nextflow.config:191]: Nevermind, it's from the nf-core template

**Review by FriederikeHanssen — APPROVED** [review]
The scripts should get licenses at some point

**Comment by jfy133** [comment]
> The scripts should get licenses at some point

Yup, Maxime pointed this out too :grimacing: - need to find original authors or come up with best way to put e.g. nf-core communtiy or something, but will take a bit of time. Made an issue already #570

**Pull request merged**